src/api/callback.cc

```diff
@@ -26,6 +26,16 @@ CallbackScope::CallbackScope(Isolate* isolate,
   try_catch_.SetVerbose(true);
 }
 
+CallbackScope::CallbackScope(Environment* env,
```

Comment:
Can you clean up the existing constructor above by making it call this one, so that we don’t duplicate all the code here? i.e.

```c++
CallbackScope::CallbackScope(Isolate* isolate,
                             Local<Object> object,
                             async_context async_context)
  : CallbackScope(Environment::GetCurrent(isolate), object, async_context) {}
```

Reply:
@addaleax Done, PTAL